uiowa/uiowa
# [Registrar] Final exams

Resolves #8077 





# How to test



```
git checkout registrar-final-exams && git pull && ddev composer install && ddev blt frontend && ddev blt ds --site=registrar.uiowa.edu && ddev drush @registrar.local uli /test-exam-schedule
```

Check out the Exam schedule, test out the search box, etc., and compare it to the existing site's: https://registrar.uiowa.edu/final-exam-schedules/fall-2024

Note: There are 2 exam blocks on this test page. In prod, we wouldn't expect needing the second, so overlap between their functionality is probably okay. Feel free to delete one.

Edit the layout and change the session, see that the page doesn't break, and has an appropriate "no results" message.

Check that other MAUI-based services still work correctly, including the session dates at the bottom of this page, the classrooms buildings importer, Academic Dates calendar (https://registrar.uiowa.ddev.site/calendars-and-deadlines/academic-calendar), and others.

(A search for something like `$this->maui` in the codebase can help reveal others to test.)
